resources/js/oe_whitelabel.project_status.js

```diff
@@ -0,0 +1,62 @@
+/**
+ * @file
+ * Attaches behaviors for the project status element.
+ */
+(function (bootstrap, Drupal, $) {
+
+  const colorClasses = [
+    'bg-secondary',
+    'bg-info',
+    'bg-dark',
+  ];
+
+  /**
+   * Animates the project status badge and progress bar.
+   *
+   * @type {Drupal~behavior}
+   *
+   * @prop {Drupal~behaviorAttach} attach
+   *   Initialises the behavior.
+   */
+  Drupal.behaviors.projectStatus = {
+    attach: function (context) {
+      $('.bcl-project-status', context).once('bcl-project-status').each(function () {
+        const $element = $(this);
+        const msBegin = $element.data('start-timestamp') * 1000;
+        const msEnd = $element.data('end-timestamp') * 1000;
+        const statusLabels = $element.data('status-labels').split('|');
+        const msNow = Date.now();
+        const status = (msNow >= msBegin) + (msNow > msEnd);
+        const progress01 = Math.max(0, Math.min(1, (msNow - msBegin) / (msEnd - msBegin)));
+        const percent = Math.round(progress01 * 200) / 2;
+
+        // Process the status label.
+        $('.badge', this).each(function () {
+          const $element = $(this);
+          $element.removeClass(colorClasses);
+          $element.addClass(colorClasses[status]);
+          if (statusLabels) {
+            $element.html(statusLabels[status]);
+          }
+        });
+
+        // Process the progress bar.
+        $('.progress-bar', this).each(function () {
+          const $element = $(this);
+          $element.removeClass(colorClasses);
+          $element.addClass(colorClasses[status]);
+          // Disable css transition.
+          // It looks bad for the initial setting, and afterwards it does not help.
+          $element.css('transition', false);
```

Comment:
I didn't notice any issue, dont think we should fiddle with these, rather report, but not a blocker, seems to work the same with or without. If you found issue create a ticket, and once fixed, remove this "fix"

Reply:
True, no longer necessary.

Reply:
The transition would only go from cached value to current value.
But it does not happen anyway because the element is hidden on page load.
So I remove this code.